apps/remix-ide/src/app.js

```diff
@@ -500,4 +505,30 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
   framingService.start(params)
 
   if (params.embed) framingService.embed()
+  if (!params) {
+    if (!localStorage.getItem('hadTour_initial')) {
+      introJs().setOptions({
+        steps: [{
+          title: 'Welcome to RemixIDE',
+          intro: 'Click on an icon to launch our Home tab with all neccessary links and tips.',
+          element: document.querySelector('#verticalIconsHomeIcon'),
+          tooltipClass: 'bg-light text-dark'
+        },
+        {
+          element: document.querySelector('#verticalIconsKindsolidity'),
+          title: 'Solidity',
+          intro: 'Select a contract and switch to solidity plugin to compile',
+          tooltipClass: 'bg-light text-dark'
+        },
+        {
+          title: 'Deploy your contract',
+          element: document.querySelector('#verticalIconsKindudapp'),
+          intro: 'Here you go, almost done. Now switch to the plugin to Deploy your contract.',
```

Comment:
Choose a chain, deploy a contract and play with your functions.

or (Haiku style)

put your contract on a chain
do not disdain
it is not much of a pain